netket/machine/abstract_machine.py

```diff
@@ -159,3 +159,7 @@ def save(self, file):
 
     def load(self, file):
         self.parameters = _np.load(file, allow_pickle=False)
+
+    @property
+    def n_visible(self):
+        return self.hilbert.size
```

Comment:
Shouldn't we change the name of this to `input_size` or something like that?
The name `n_visible` is really something that only makes sense for RBMs...

Reply:
yes, this was mostly for backward compatibility